Index: src/main/resources/application.conf
===================================================================
diff --git a/src/main/resources/application.conf b/src/main/resources/application.conf
--- a/src/main/resources/application.conf	
+++ b/src/main/resources/application.conf	
@@ -1,9 +1,20 @@
 ktor {
+  development = true
+
   deployment {
     port = 8080
     port = ${?PORT}
+  //  watch = [killua]
+    shutdown.url = "/katyIloveYou"
+
   }
   application {
     modules = [com.killua.ApplicationKt.module]
   }
+  database {
+    DatabaseUrl = "jdbc:postgresql://gk-coding.com:5432/transy_db"
+    DatabaseDriver = "org.postgresql.Driver"
+    DatabaseUser = "killuapgadmin"
+    DatabasePassword = "t72y3ypFShxdro"
+  }
 }
